tests/unit/dojson/test_dojson_hep_bd9xx.py

```diff
@@ -557,3 +559,67 @@ def test_references_from_999C50_h_m_o_r_y():
     result = hep2marc.do(result)
 
     assert expected == result['999C5']
+
+
+@pytest.mark.xfail
```

Comment:
please provide a `reason` to the `xfail` (like `xfail(reason=Multiple report numbers are not handled)`)